src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -334,6 +490,38 @@ public static MvcForm BeginRouteForm(this IHtmlHelper htmlHelper, string routeNa
                 routeName,
                 routeValues: null,
                 method: FormMethod.Post,
+                suppressAntiforgery: false,
+                htmlAttributes: null);
+        }
+
+        /// <summary>
+        /// Renders a &lt;form&gt; start tag to the response. The route with name <paramref name="routeName"/>
+        /// generates the &lt;form&gt;'s <c>action</c> attribute value.
+        /// </summary>
+        /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
+        /// <param name="routeName">The name of the route.</param>
+        /// <param name="suppressAntiforgery">
+        /// If <c>true</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token. By
+        /// default &lt;form&gt; tags will automatically include an antiforgery token.
```

Comment:
Minor but `&lt;form&gt;` tags include no such token. You mean `&lt;form&gt;` elements.